vnext/ReactUWP/Base/UwpReactInstance.cpp

```diff
@@ -206,7 +202,7 @@ void UwpReactInstance::Start(const std::shared_ptr<IReactInstance> &spThis, cons
           devSettings->jsiRuntimeHolder = std::make_shared<facebook::react::V8JSIRuntimeHolder>(
               devSettings, jsQueue, std::move(scriptStore), std::move(preparedScriptStore));
           break;
-#endif
```

Comment:
I think this may be removing the wrong #endif

Reply:
Good catch! I will fix this.